### PR TITLE
add instructions on how to work with fork repos and fix some of the m…

### PR DESCRIPTION
**Pull request by uturunku1**
…arkdown lint issues

# Issue

# Goals

# Implementation Details

# How to Test

### CODE REVIEW

File: docs/Contributing.md [hunk 1]
```diff
@@ -14,9 +14,14 @@ Before submitting an Issue, please search for similar ones in the
...
-3. You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+1. Fork this repo and clone your forked version of this repo
+2. When you create branch, name it after your Github Handle followed by a slash, and assign it a descriptive name of the changes you will make. Examples:
+`git branch DanaScully/update-notempirical-docs`
+`git branch fMulder/create-BaseEnvFindTruth-task`
+`git branch TheLoneGunmen/issue29-fixfalsepositive` (assuming this contributor is addressing one of our github issues)
+3. Once you open a Pull Request, sign the License Agreement which will show up in the UI as one of our checks. We'll talk in more detail about this in the next section.
+4. After the PR check have ran, if you notice the banner `this branch is out-of-date with the base branch`, get the latest changes from upstream into your origin branch (you can find some recommended steps on how to do this at the bottom of this doc.)
```
Comment (GabrielSamp) [inline docs/Contributing.md:23]: missing 's' on check

**Review by GabrielSamp — CHANGES_REQUESTED** [review]
please add an 's' on the end of "PR check" on line 23 of contributing.md doc :) Other then that looks awesome!!!

File: docs/Contributing.md (hunk 1 above)
Comment (uturunku1) [reply docs/Contributing.md:23]: Adding that S👍

**Review by GabrielSamp — APPROVED** [review]
Looks good :)

**Pull request merged**